files/en-us/learn/getting_started_with_the_web/html_basics/index.html

```diff
@@ -157,7 +157,7 @@ <h3 id="Paragraphs">Paragraphs</h3>
 
 <pre class="brush: html notranslate">&lt;p&gt;This is a single paragraph&lt;/p&gt;</pre>
 
-<p>Add your sample text (you should have it from <a href="/en-US/docs/Learn/Getting_started_with_the_web/What_should_your_web_site_be_like"><em>What should your website look like?</em></a>) into one or a few paragraphs, placed directly below your {{htmlelement("img")}} element.</p>
+<p>Add your sample text (you should have it from <a href="/en-US/docs/Learn/Getting_started_with_the_web/What_will_your_website_look_like"><em>What should your website look like?</em></a>) into one or a few paragraphs, placed directly below your {{htmlelement("img")}} element.</p>
```

Comment:
To match resulting updated URL
```suggestion
<p>Add your sample text (you should have it from <a href="/en-US/docs/Learn/Getting_started_with_the_web/What_will_your_website_look_like"><em>What will your website look like?</em></a>) into one or a few paragraphs, placed directly below your {{htmlelement("img")}} element.</p>
```